### ISSUE TITLE
Facebook API

### ISSUE THREAD

**Issue by william21101** [william21101/HANDZ#1]

**Comment by gtenorio** [comment]
Curious, what are you using for your Git environment? Windows?

**Comment by william21101** [comment]
Yeah I'm using my little Mac computer to code everything. Still a little unsure about what a git exactly is...

**Comment by rachan5** [comment]
Git is version control for source code (manly). 

Think of it like Google Docs, but for code. Multiple people can edit, and you can jump around from different versions of a essay/chunk of code, but its all in one file/place.

Good old Mr. Simon has a nice assignment about it. Does this count as a throw back? It didn't exist when we took AP Computer Science....

[(https://github.com/APCSLowell/SampleAssignment)]

**Comment by Grimstad** [comment]
The use of Facebook as a front-end with Jango is entirely consistent with our desired client-db. This could be the easiest way to reach a billion players. (I love this Git program already.)
